tasks/drupal/deploy_database.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+- name: Download project db dump
```

Comment:
It should support additional headers like the steamengine_build_url_headers does for the build url.